tests/foreman/ui/test_contentview.py

```diff
@@ -621,17 +619,18 @@ def test_positive_publish_rh_content_with_errata_by_date_filter(self):
         })
         RepositorySet.enable({
             'basearch': 'x86_64',
-            'name': REPOSET['rhst6'],
+            'name': REPOSET['rhva6'],
```

Comment:
As per https://bugzilla.redhat.com/show_bug.cgi?id=1455990#c2 , satellite tools is an exclusion from test scenario so it should be replaced with other repo. Comment suggests rhel repo, but its sync will take much more time, rhav6 repo has packages with no associated errata as well and its sync is way faster.